lib/functions/versions.bash

```diff
@@ -14,13 +14,17 @@ version_command() {
   shift 2
   local versions=("$@")
 
+  local file_name
   local file
+
+  file_name="$(version_file_name)"
+
   if [ "$cmd" = "global" ]; then
-    file=${ASDF_DEFAULT_TOOL_VERSIONS_FILENAME:-$HOME/.tool-versions}
```

Comment:
Rather than having this "if custom file name use it, else use `.tool-versions`" logic spread around through the code, I have introduced a `version_file_name` helper function.